r2/r2/lib/base.py

```diff
@@ -104,7 +107,7 @@ def __call__(self, environ, start_response):
             request.ip = cdn_ip
             request.via_cdn = True
         elif g.trust_local_proxies and forwarded_for and is_local_address(remote_addr):
-            request.ip = forwarded_for.split(',')[-1]
+            request.ip = forwarded_for.split(',')[0].strip()
```

Comment:
This reads from the opposite end of the list of forwarded-fors, is that intended?